src/components/timeout-dialog/timeout-dialog.yaml

```diff
@@ -48,6 +48,7 @@ examples:
     keepAliveButtonText: "keepAliveButtonText"
     signOutButtonText: "signOutButtonText"
     synchroniseTabs: false
+    hideSignOutLink: false
```

Comment:
we could add a comment here maybe, `this is experimental - don't use without testing, added at request of non-hmrc service, where a service only has an anonymous session, so no explicit log in action has been taken, and you might want to have no early sign out link rather along with the than a link saying something like [abandon attempt]`